lua/lspconfig/configs/tinymist.lua

```diff
@@ -7,6 +77,7 @@ return {
     end,
     single_file_support = true,
   },
+  commands = tinymist_commands,
```

Comment:
No. Just use the normal nvim_create_user_command api in a on_attach. This old 'commands' thing is deprecated.